tools/gnali/gnali.xml

```diff
@@ -0,0 +1,73 @@
+<tool id="gnali" name="Get nonessential, LoF variants" version="0.1.0" python_template_version="3.6">
```

Comment:
I would change name to gNALI.

You can can a `<decscription></description>` section just inside and put what you have currently in the name field.

Reply:
2a7404f Changed name to "gNALI" and added description